tests/auth.test.js

```diff
@@ -0,0 +1,135 @@
+import chai, { should } from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import expressValidator from 'express-validator';
+
+import app from '../src';
+
+import {
+  shortFirstName,
+  undefinedFirstName,
+  invalidFirstName,
+  shortLastName,
+  undefinedLastName,
+  invalidLastName,
+  undefinedEmail,
+  undefinedUsername,
+  shortUsername,
+  invalidEmail,
+  undefinedPassword,
+  shortPassword,
+  invalidPassword,
+  validUser
+} from './mockData/mockUser';
```

Comment:
Please change the `mockData` folder to the `fixtures` folder. Tests fixtures sound better than `mockData`.